airflow/providers/datadog/sensors/datadog.py

```diff
@@ -39,15 +41,15 @@ class DatadogSensor(BaseSensorOperator):
     @apply_defaults
     def __init__(
             self,
-            datadog_conn_id='datadog_default',
-            from_seconds_ago=3600,
-            up_to_seconds_from_now=0,
-            priority=None,
-            sources=None,
-            tags=None,
+            datadog_conn_id: str = 'datadog_default',
+            from_seconds_ago: int = 3600,
+            up_to_seconds_from_now: int = 0,
+            priority: Optional[str] = None,
+            sources: Optional[List[Any]] = None,
+            tags: Optional[List[Any]] = None,
             response_check=None,
```

Comment:
@mik-laj @potiuk @OmairK What is the type of `response check` ?

Reply:
Seems like a [callable](https://docs.python.org/3/library/typing.html#callable). What do you think?

Reply:
i was thinking maybe `bool`